packages/react/src/index.ts

```diff
@@ -19,6 +20,13 @@ import { Effect, ReactDispatcher } from "./internal";
 
 export { signal, computed, batch, effect, Signal, type ReadonlySignal };
 
+const Empty = [] as const;
+
+/**
+ * React uses a different entry-point depending on NODE_ENV env var
+ */
+const __DEV__ = process.env.NODE_ENV !== "production";
```

Comment:
We'll need to find a way to ignore the double effect thing without relying on this - it breaks in ESM.